airbyte-integrations/connectors/source-iterable/source_iterable/source.py

```diff
@@ -66,6 +66,7 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
         # end date is provided for integration tests only
         start_date, end_date = config["start_date"], config.get("end_date")
         date_range = {"start_date": start_date, "end_date": end_date}
+        region = config["region"]
```

Comment:
```suggestion
        region = config["region"] or "US"
```

This way it won't break existing connections.